requirements.txt

```diff
@@ -1,4 +1,6 @@
 -e git+http://github.com/xamox/pygame
 MySQL-python==1.2.5
-Django==1.6.6
+Django==1.7
```

Comment:
1.7 is not secure anymore. I wonder if we can upgrade directly to 1.9 without causing any issues. If we can, I think we should!